src/data/cache.ts

```diff
@@ -10,7 +10,10 @@ import { typed_keys } from "../misc-functions/third_party/typed-keys";
 import { WorkspaceSecurity } from "../types";
 import { Cacheable } from "./types";
 
-const cacheFolder = path.join(process.cwd(), "cache");
+if (!process.env.MCFUNCTION_CACHE_DIR) {
+    throw new Error("Environment variable MCFUNCTION_CACHE_DIR must be set");
+}
+const cacheFolder = process.env.MCFUNCTION_CACHE_DIR;
```

Comment:
There should also be a way to not have to set an environment variable & maybe pass the dir in as an argument

Reply:
I don't think this is a major concern, as only extension authors would need to pass the environment variable.